recipe/meta.yaml

```diff
@@ -63,22 +83,25 @@ outputs:
 
   - name: sktime-all-extras
     requirements:
+      host:
+        - python
       run:
         - python
-        - {{ pin_subpackage("sktime", max_pin="x.x.x.x") }}
-        - matplotlib-base >=3.3.2
-        - pmdarima >=1.8.0,!=1.8.1  # [not win and py<=38]
-        - scikit-posthocs >=0.6.5
-        - seaborn >=0.11.0
-        - tsfresh >=0.17.0
-        - numba >=0.53
-        # - catch22 >=0.1.0
-        - stumpy >=1.5.1
-        - tbats >=1.1.0
-        - prophet >=1.1  # [not aarch64 and not linux and py>37]
-        - filterpy >=1.4.5
-        - pykalman >=0.9.5
+        - {{ pin_subpackage("sktime", exact=True) }}
+        - matplotlib-base
+        - pmdarima  # [not win and py<=38]
+        - prophet   # [not aarch64 and not linux and py>37]
```

Comment:
@lmmentel, could you please confirm/explain why these selectors are needed?

Reply:
I have already removed the `pmdarima` selector because I think it is not needed and updated the prophet selector with current availability.